parkhangs-webapp/src/features/parks/parksSlice.js

```diff
@@ -1,12 +1,30 @@
-import {createSlice} from '@reduxjs/toolkit'
+import {createSlice, createAsyncThunk} from '@reduxjs/toolkit'
+import axios from 'axios';
+
+//Thunk crated here
+export const fetchEventsById = createAsyncThunk(
+    'parks/fetchEventsByIdStatus', (id) => {
+        axios
```

Comment:
I find async/await logic is easier to read, and also follows the conventions from the `createAsyncThunk` page you sent us https://redux-toolkit.js.org/api/createAsyncThunk
```
    async (id, thunkAPI) => {
        const response = await axios.get(`http://localhost:9000/${id}/events`)
        return response.data
    }
```

and then you can use the action types from that page to handle failures/loading states

Reply:
Changed and noted :)